daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -325,22 +307,65 @@ private[lf] object Speedy {
       implicit val loggingContext: LoggingContext,
       /* Compiled packages (Daml-LF ast + compiled speedy expressions). */
       var compiledPackages: CompiledPackages,
```

Comment:
`compiledPackages` will require changes to the class `CompiledPackages` so that we can better control how its read/write